tests/phpunit/tests/dependencies/styles.php

```diff
@@ -628,4 +628,54 @@ public function test_wp_maybe_inline_styles_no_path() {
 
 		$this->assertSame( $GLOBALS['wp_styles']->registered['test-handle']->src, $url );
 	}
+
+	/**
+	 * Tests detecting nested tags in inline styles.
+	 *
+	 * @ticket 61828
+	 * @dataProvider data_styles_with_nested_tags
```

Comment:
```suggestion
	 * @dataProvider data_wp_add_inline_style_with_nested_style_tags
```

Convention is for data provideres to use the same name with test replaced by data